packages/reports/src/Stores/Modules/Profit/profit-store.js

```diff
@@ -69,13 +69,12 @@ export default class ProfitTableStore extends BaseStore {
         return !!(this.date_from || this.date_to);
     }
 
-    shouldFetchNextBatch() {
-        if (this.has_loaded_all || this.is_loading) return false;
-        return true;
+    shouldFetchNextBatch(is_mounting) {
+        return !!is_mounting || (!this.has_loaded_all && !this.is_loading);
     }
 
-    async fetchNextBatch(shouldFilterContractTypes) {
-        if (!this.shouldFetchNextBatch()) return;
+    async fetchNextBatch(shouldFilterContractTypes, isMounting) {
```

Comment:
Do we need to pass second arg in packages/trader/src/AppV2/Containers/Positions/positions-content.tsx where this function is used?